src/libs/Navigation/types.ts

```diff
@@ -620,6 +620,7 @@ type WorkspacesCentralPaneNavigatorParamList = {
 type FullScreenNavigatorParamList = {
     [SCREENS.WORKSPACE.INITIAL]: {
         policyID: string;
+        enabledFeatureRouteName?: string;
```

Comment:
```suggestion
        enabledFeature?: string;
```

Reply:
Should we make this more strict and only allow the array of feature names here instead of `string`?

Reply:
In what situation would we have an array of string? Currently, we redirect when a feature is toggled right away.

Reply:
Ah, I get you now. Will add this.

Reply:
This is not needed since we moved to focus effect.